src/app/user/user.service.ts

```diff
@@ -60,33 +63,33 @@ export class UserService {
     this.searchUrl = apiUrl + 'search';
     // TODO - switch to internal observable that is populated on initialization
     // and only expose currentLoggedInUser publicly
-    this.loggedInUser = Observable.merge(
+    this.loggedInUser = merge(
       broadcaster.on('loggedin')
-        .map(val => 'loggedIn'),
+        .pipe(map((val: any) => 'loggedIn')),
```

Comment:
Can you not simply remove val?
`.pipe(map(() => 'loggedIn'))`

Reply:
good point